src/main/java/frc/robot/Constants.java

```diff
@@ -307,8 +307,19 @@ public static final class ScoringConstants {
         public static final double shooterAcceleration = 1;
         public static final double shooterJerk = 1;
 
-        public static final double shooterVelocityRPMMargin = 0;
-        public static final double aimAngleRadiansMargin = 0;
+        public static final double shooterVelocityRPMMargin = 10;
+        public static final double aimAngleRadiansMargin = Math.PI / 100;
```

Comment:
This would be a good time to add a `kDegreesToRadians` constant and put the margin in degrees * kDegreesToRadians (and also in the below margin for hood angle)

Reply:
I could use the Units library